roles/core/log_client/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 log_client_server_port: "{{ log_server_port | default(514) }}"
 log_client_verbosity: "info"
+log_client_configuration_files: []
```

Comment:
This is unnecessary to declare this one as default, this is an optional parameter.